eng/scripts/Test-Packages.ps1

```diff
@@ -46,6 +48,17 @@ $env:RUSTFLAGS = "-Dwarnings"
 foreach ($package in $packagesToTest) {
   Push-Location ([System.IO.Path]::Combine($RepoRoot, $package.DirectoryPath))
   try {
+    $serviceDirectory = ([System.IO.Path]::Combine($RepoRoot, $package.DirectoryPath)) + "/../"
+    Write-Host "Checking for setup in $serviceDirectory + 'Test-Setup.ps1'"
+    if (Test-Path ($serviceDirectory + "Test-Setup.ps1")) {
```

Comment:
Use `[System.IO.Path]::Combine()` for joining paths. Never use `join-path`, and don't assume your base is correctly terminated.